src/software/backend/radio_backend.cpp

```diff
@@ -6,7 +6,7 @@
 #include "software/util/design_patterns/generic_factory.h"
 
 RadioBackend::RadioBackend(
-    std::shared_ptr<const SSLCommunicationConfig> ssl_communication_config)
+    std::shared_ptr<const SslCommunicationConfig> ssl_communication_config)
```

Comment:
The name "SslCommunicationConfig" is a bit awkward. is there a way to keep it the same as before?

Reply:
Its actually a convention to treat acronyms as words and use camelCase which makes things much easier to work with. Jason mentioned hes seen this convention in use before

We would need to create exceptions for `Ai, Ssl, Id`, its _slightly_ annoying to create these exceptions but we can do it.

https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

Thoughts @itsarune  @jonathanlew  @tarakong01 @jsn0716 @kelvinkoon ?

Reply:
ATM, I prefer the camelCase b/c the other similar configurations (eg. `NetworkConfig` and `SensorFusionConfig`) are also camelCase'd.

Reply:
I definitely prefer the convention of treating acronyms as words. Consider a case where you have to write an acronym at the beginning in camel case with first letter lower case, where keeping the acronyms in upper case conflicts with the first letter lower case style. While this case doesn't specifically apply here, I would prefer the styles be kept consistent.

Reply:
According to the SO post, two-letter abbreviations should be all-caps. I'm OK with that convention. So it would be, (pascal case/camel case) AI/AI, Ssl/ssl, ID/ID.

Reply:
Consistency makes sense, so keeping it is good